keps/sig-node/3953-dynamic-node-resize/README.md

```diff
@@ -0,0 +1,776 @@
+     |                                |            poll               |                                |
+     |                                |------------------------------>|                                |
+     |                                |                               |                                |
+     |                                |                               |                                |
+     |                                |                               |             fetch              |
+     |                                |                               |------------------------------->|
+     |                                |                               |                                |
+     |                                |                               |                                |
+     |                                |                               |                                |
+     |                                |                               |             update             |
+     |                                |                               |<-------------------------------|
+     |                                |                               |                                |
+     |                                |            update             |                                |
+     |                                |<------------------------------|                                |
+     |                                |                               |                                |
+     |                                |                               |                                |
+     |                                |                               |                                |
+     |     node status update         |                               |                                |
+     |<-------------------------------|                               |                                |
+     |                                |                               |                                |
+     |      if shrink in resource     |                               |                                |
+     |      re-run pod admission      |                               |                                |
+     |<-------------------------------|                               |                                |                                |
+     |                                |                               |                                |      
+     | re-initialize resource managers|                               |                                |
+     |<-------------------------------|                               |                                |                                |
+     |                                |                               |                                |           
+
+```
```

Comment:
BTW, you can use Mermaid in GitHub - see https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/

Reply:
Thanks for the reference, I will try to use that.